packages/core/test/helpers.ts

```diff
@@ -12,7 +12,10 @@ export function nullLogger(): Logger {
 }
 
 export class TestTransport implements Transport {
-  send(_options: TransportOptions, _payload: NoticeTransportPayload): Promise<{ statusCode: number; body: string }> {
+  defaultHeaders(): Record<string, string> {
+    return {};
+  }
+  send<T>(_options: TransportOptions, _payload: T): Promise<{ statusCode: number; body: string }> {
```

Comment:
Made the `payload` to be generic because I didn't want to define the payload for check-ins here (check-ins are not part of the `core` package)